File: Java/Miscellaneous/DeliveryType.java
package LTD_API;

public class DeliveryType
{
    private int deliveryTypeId;
    private String deliveryTypeName;
    private decimal deliveryTypePrice;
    
    public int getDeliveryTypeId()
    {
        return deliveryTypeId;
    }
    public void setDeliveryTypeId(int deliveryTypeId)
    {
        this.deliveryTypeId = deliveryTypeId;
    }

    public String getDeliveryTypeName()
    {
        return deliveryTypeName;
    }
    public void setDeliveryTypeName(String deliveryTypeName)
    {
        this.deliveryTypeName = deliveryTypeName;
    }

    public decimal getDeliveryTypePrice()
    {
        return deliveryTypePrice;
    }
    public void setDeliveryTypePrice(decimal deliveryTypePrice)
    {
        this.deliveryTypePrice = deliveryTypePrice;
    }
}

File: PHP/Miscellaneous/DeliveryType.php
<?php
namespace LTD_API;

class DeliveryType
{
    private $deliveryTypeId;
    private $deliveryTypeName;
    private $deliveryTypePrice;
    
    public function getDeliveryTypeId()
    {
        return $this->deliveryTypeId;
    }
    public function setDeliveryTypeId($deliveryTypeId)
    {
        $this->deliveryTypeId = $deliveryTypeId;
    }

    public function getDeliveryTypeName()
    {
        return $this->deliveryTypeName;
    }
    public function setDeliveryTypeName($deliveryTypeName)
    {
        $this->deliveryTypeName = $deliveryTypeName;
    }

    public function getDeliveryTypePrice()
    {
        return $this->deliveryTypePrice;
    }
    public function setDeliveryTypePrice($deliveryTypePrice)
    {
        $this->deliveryTypePrice = $deliveryTypePrice;
    }
}

